### ISSUE TITLE
Build fails when using RenderErrorPage

### ISSUE THREAD

**Issue by nbrdx** [brillout/vite-plugin-ssr#717]
## Problem
The build fails when using `RenderErrorPage` in an `*.page.route.ts` file.

![image](https://user-images.githubusercontent.com/10615700/225965298-820d0113-798e-4872-a6ea-b0743dc774cc.png)

The file is simple

![image](https://user-images.githubusercontent.com/10615700/225965881-0a5f96b5-32df-43b7-b6d7-17da3419994a.png)

It looks like I can import RenderErrorPage normally but cannot build my app with it.

## Stack
Using Vite + vite-plugin-ssr + React + TS + Ts.ed

### Error Message + Error Stack

```shell
'RenderErrorPage' is not exported by ../../../node_modules/vite-plugin-ssr/dist/esm/client/node.js, imported by pages/error.page.route.ts
file: /Volumes/Develop/clubmed/digital-oidc/packages/gm/app/pages/error.page.route.ts:2:9
1: import { PageContext } from "@clubmed/components";
2: import { RenderErrorPage } from "vite-plugin-ssr";
            ^
3: 
4: export default (pageContext: PageContext) => {
error during build:
Error: 'RenderErrorPage' is not exported by ../../../node_modules/vite-plugin-ssr/dist/esm/client/node.js, imported by pages/error.page.route.ts
    at error (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:1858:30)
    at Module.error (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:12429:16)
    at Module.traceVariable (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:12788:29)
    at ModuleScope.findVariable (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:11440:39)
    at ReturnValueScope.findVariable (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
    at ChildScope.findVariable (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
    at TrackingScope.findVariable (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
    at BlockScope.findVariable (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
    at Identifier.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:7439:40)
    at CallExpression.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5269:23)
    at CallExpression.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:8935:15)
    at ThrowStatement.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5269:23)
    at BlockStatement.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5265:73)
    at IfStatement.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5269:23)
    at BlockStatement.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5265:73)
    at ArrowFunctionExpression.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5269:23)
    at ExportDefaultDeclaration.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5269:23)
    at Program.bind (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:5265:73)
    at Module.bindReferences (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:12425:18)
    at Graph.sortModules (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:23062:20)
    at Graph.build (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:22940:14)
    at async file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:23730:13
    at async catchUnfinishedHookActions (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:23126:20)
    at async rollupInternal (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/rollup/dist/es/shared/rollup.js:23727:5)
    at async doBuild (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/vite/dist/node/chunks/dep-c6273c7a.js:43490:24)
    at async build (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/vite/dist/node/chunks/dep-c6273c7a.js:43330:16)
    at async CAC.<anonymous> (file:///Volumes/Develop/clubmed/digital-oidc/node_modules/vite/dist/node/cli.js:747:9)

Process finished with exit code 1
```

**Comment by brillout** [comment]
It's because `RenderErrorPage` currently only works on the server-side. (I.e. `.page.server.js`, or `.page.route.js` if you use Server Routing.)

There has been internal changes which should make it easy to implement `RenderErrorPage` on the client-side. I'll have a look at it later today.

**Comment by brillout** [comment]
Released in `0.4.97`. Make sure to replace your import:

```diff 
- import { RenderErrorPage } from 'vite-plugin-ssr'
+ import { RenderErrorPage } from 'vite-plugin-ssr/RenderErrorPage'

**Comment by brillout** [comment]
Also, in case you think you can convince your company: https://github.com/sponsors/brillout.

**Comment by nbrdx** [comment]
Thank you for your reactivity !
Works in `0.4.99` but TS (4.9.4) seems to throw an error when importing like this
![image](https://user-images.githubusercontent.com/10615700/226857513-5e472e79-1810-48b3-ac8a-390eafa77355.png)
To make it pass, I need to `@ts-ignore` it

**Comment by brillout** [comment]
Indeed. Let me fix this.

**Comment by brillout** [comment]
Fix released in `0.4.100`.

**Comment by nbrdx** [comment]
Thank you, working fine 🥳
